backend/compact-connect/lambdas/python/data-events/tests/function/test_encumbrance_events.py

```
@@ -704,3 +807,1593 @@ def test_license_encumbrance_lifted_listener_does_not_unencumber_when_license_re
+        self.test_data_generator.put_default_privilege_record_in_provider_table(
+            value_overrides={
+                'jurisdiction': 'ky',
+                'administratorSetStatus': 'active',
+            }
+        )
+
+        message = self._generate_privilege_encumbrance_message()
+        event = self._create_sqs_event(message)
+
+        # Execute the handler
+        result = privilege_encumbrance_notification_listener(event, self.mock_context)
```

Pylint (3.3.7)

[error] Too many positional arguments for function call

(E1121)